core-java-modules/core-java-collections-array-list-2/src/main/java/com/baeldung/list/multiple/objecttypes/AlternativeMultipeTypeList.java

```diff
@@ -0,0 +1,51 @@
+package com.baeldung.list.multiple.objecttypes;
```

Comment:
Let's use a less deeply nested package structure. Name it com.baeldung.list.multipleobjecttypes.